### ISSUE TITLE
Linux actions build failing due to missing/incorrect javac

### ISSUE THREAD

**Issue by joeshannon** [DiamondLightSource/dawn-cbflib-build#2]
The [actions build](https://github.com/DiamondLightSource/dawn-cbflib-build/actions/workflows/build-java-linux.yml) for Linux (+ Windows) is currently failing due to:

```
 /usr/lib/jvm/bin/javac -d . /io/cbflib/jcbf/*.java
bash: /usr/lib/jvm/bin/javac: No such file or directory
make: *** [/io/cbflib/jcbf/cbflib-0.9.7.jar] Error 127
Error: Process completed with exit code 2.
```

**Comment by joeshannon** [comment]
`$JBIN` is being set to `JBIN=/usr/lib/jvm/java-11-openjdk-11.0.20.0.8-1.el7_9.x86_64/bin/java` then the three end path segments chopped off with multiple `dirname`s. 

Looks like we only need to remove the two parts now.

Is this due to a change in the image used @PeterC-DLS ?

**Comment by PeterC-DLS** [comment]
I need to update the build scripts as the runners have changed.

**Comment by PeterC-DLS** [comment]
Hopefully fixed by [a939c1a39](https://github.com/DiamondLightSource/dawn-cbflib-build/commit/a939c1a39913203d83ec97161cc05e0ae7824e97)

**Comment by PeterC-DLS** [comment]
Further fix in [e7b3c4bc0](https://github.com/DiamondLightSource/dawn-cbflib-build/commit/e7b3c4bc09b76d34142d31ea9c893a00ea0e3514)
